core-experimental/include/iDynTree/Core/PositionRaw.h

```diff
@@ -8,86 +8,47 @@
 #ifndef IDYNTREE_POSITION_RAW_H
 #define IDYNTREE_POSITION_RAW_H
 
-#include "IVector.h"
+#include <iDynTree/Core/IVector.h>
+#include <iDynTree/Core/VectorFixSize.h>
 #include <string>
 
 namespace iDynTree
 {
     class RotationRaw;
-    
+
     /**
      * Class providing the raw coordinates for iDynTree::Position class.
      *
      * \ingroup iDynTreeCore
      */
-    class PositionRaw: public IVector
+    class PositionRaw: public Vector3
```

Comment:
One alternative (depending if you want to avoid the user uses "low level" methods) is to let the `PositionRaw` to use a `Vector3` instead to be a `Vector3`